docs/development.rst

```diff
@@ -158,66 +141,66 @@ handled.
 
 Adding a Theory to the codebase is done by following these steps:
 
-1. Tests: Add a test file ```pysmt/test/test_<theory>.py```, to
-demonstrate the use for the theory (e.g., ```pysmt/test/test_bv.py```).
+1. Tests: Add a test file ``pysmt/test/test_<theory>.py``, to
+    demonstrate the use for the theory (e.g., ``pysmt/test/test_bv.py``).
 
-2. Operators: Add the (basic) operators in ```pysmt/operators.py```,
-create a constant for each operator, and extend the relevant
-structures.
+2. Operators: Add the (basic) operators in ``pysmt/operators.py``,
+    create a constant for each operator, and extend the relevant
+    structures.
 
-3. Typing: Extend ```pysmt/typing.py``` to include the types (sorts)
-of the new theory.
+3. Typing: Extend ``pysmt/typing.py`` to include the types (sorts)
+    of the new theory.
 
-4. Walker: Extend ```pysmt/walkers/generic.py``` to include one
-```walk_``` function for each of the basic operators.
+4. Walker: Extend ``pysmt/walkers/generic.py`` to include one
+    ``walk_`` function for each of the basic operators.
 
-5. FNode: Extend ```is_*``` methods in
-```pysmt/fnode.py:FNode```. This makes it possible to check the type
```

Comment:
In order for the link to work, you cannot use the verbatim syntax. See Line 86, where there is a link to the TestCase class.